CONTRIBUTION.MD

```diff
@@ -0,0 +1,29 @@
+# Running Tests
+
+To ensure that everything still works, run the integration tests in the example folder.
+
+## **Running using flutter devices**
+
+Use the command:
+
+```
+flutter test integration_test/app_test.dart
+```
+
+If multiple devices was found, select the device you want to run the tests on.
+
+## **Running using command line**
```

Comment:
What do you mean with this? The action above is also using command line, maybe we need to change this to something more descriptive.

Reply:
Thanks for pointing it out. I have made the changes.